core/modules/node/node.types.inc

```diff
@@ -287,12 +287,27 @@ function node_type_form($form, &$form_state, $type = NULL) {
   );
 
   $form['actions'] = array('#type' => 'actions');
+  // If this is a new content type and the Field UI module is enabled, add a
+  // 'Save and add fields' button.
+  if ($type->module === 'node' && empty($type->type) && module_exists('field_ui')) {
+    $form['actions']['add_fields'] = array(
```

Comment:
Fair enough about the change of the name of the form element. Reverted that to `save_continue`. I also added a cancel link while at it.

I've filed https://github.com/backdrop/backdrop-issues/issues/2272 as a separate issue for @quicksketch's suggestion about adding `TRUE` to the `element_children` function as that seems to be a generic fix to the logic that adds the primary class to the form button with the smallest weight value. Once that is fixed, then my previous PR in https://github.com/backdrop/backdrop/pull/1607 (which I already reopened) will just work.

Leaving this PR open as an alternative.